.github/workflows/REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,712 @@
+### Compound ifs
+
+In general, separate `if` statements should be preferred over a compound condition. This makes debugging code easier.
+
+
+```php
+// Good
+if (! $conditionA) {
+   return;
+}
+
+if (! $conditionB) {
+   return;
+}
+
+if (! $conditionC) {
+   return;
+}
+
+// do stuff
+```
+
+```php
+// bad
+if ($conditionA && $conditionB && $conditionC) {
+  // do stuff
+}
+```
+
+
+
+## Comments
+
+Comments should be avoided as much as possible by writing expressive code. If you do need to use a comment, format it like this:
+
+```php
+// There should be a space before a single line comment.
+
+/*
+ * If you need to explain a lot you can use a comment block. Notice the
+ * single * on the first line. Comment blocks don't need to be three
+ * lines long or three characters shorter than the previous line.
+ */
+```
+
+## Whitespace
+
+Statements should have to breathe. In general always add blank lines between statements, unless they're a sequence of single-line equivalent operations. This isn't something enforceable, it's a matter of what looks best in its context.
+
+```php
+// Good
+public function getPage($url)
+{
+    $page = $this->pages()->where('slug', $url)->first();
+
+    if (! $page) {
+        return null;
+    }
+
+    if ($page['private'] && ! Auth::check()) {
+        return null;
+    }
+
+    return $page;
+}
+
+// Bad: Everything's cramped together.
+public function getPage($url)
+{
+    $page = $this->pages()->where('slug', $url)->first();
+    if (! $page) {
+        return null;
+    }
+    if ($page['private'] && ! Auth::check()) {
+        return null;
+    }
+    return $page;
+}
+```
+
+```php
+// Good: A sequence of single-line equivalent operations.
+public function up()
+{
+    Schema::create('users', function (Blueprint $table) {
+        $table->increments('id');
+        $table->string('name');
+        $table->string('email')->unique();
+        $table->string('password');
+        $table->rememberToken();
+        $table->timestamps();
+    });
+}
+```
+
+Don't add any extra empty lines between `{}` brackets.
+
+```php
+// Good
+if ($foo) {
+    $this->foo = $foo;
+}
+
+// Also Good
+if ($foo) $this->foo = $foo
+
+// Bad : If only 1 line, then whitespace can be avoided
+if ($foo) {
+
+    $this->foo = $foo;
+
+}
+
+// Good: When more than 1 line statement use whitespace to group similar things together
+if ($condition) {
+
+    $var = 1; 
+    $invoice = $thie->get();
+
+    $invoice->applyCoupom();
+    $invoice->doTotalCheck();
+
+    $this->methodB();
+    $this->methodB();
+    
+    return $response;
+}
+
+```
+
+## Configuration
+
+Configuration files must use kebab-case.
+
+```
+config/
+  pdf-generator.php
+```
+
+Configuration keys must use snake_case.
+
+```php
+// config/pdf-generator.php
+return [
+    'chrome_path' => env('CHROME_PATH'),
+];
+```
+
+Avoid using the `env` helper outside of configuration files. Create a configuration value from the `env` variable like above.
+
+## Artisan commands
```

Comment:
Avoid using `env` helper outside of configuration files. Instead, use the `config` helper to retrieve configuration values.